app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -94,6 +94,10 @@ def available_resource_pools(opts = {})
       resource_pools({ :cluster_id => opts[:cluster_id] })
     end
 
+    def available_images
+      client.templates
```

Comment:
@unorthodoxgeek ping? @ohadlevy I'd say merge and open a new issue to improve it, we'd rather have this than not. Not sure whether we should put a timeout here if it can take this long too..